### PR TITLE
Fixed loading metadata for non-persistent classes

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Comment by szymach** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open

File: lib/FSi/DoctrineExtensions/Uploadable/Keymaker/Entity.php [hunk 1]
```diff
@@ -22,6 +24,8 @@ public function createKey($object, $property, $id, $originalName, $pattern = nul
             $pattern = self::DEFAULT_PATTERN;
         }
 
+        $className = ClassUtils::getRealClass(get_class($object));
```
Comment (rn0) [inline lib/FSi/DoctrineExtensions/Uploadable/Keymaker/Entity.php]: `ClassUtils::getClass($object);`

**Comment by JarekW** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Pull request merged**